src/vs/editor/common/core/characterClassifier.ts

```diff
@@ -69,11 +72,17 @@ export class CharacterSet {
 		this._actual = new CharacterClassifier<Boolean>(Boolean.False);
 	}
 
-	public add(charCode: number): void {
-		this._actual.set(charCode, Boolean.True);
+	public add(char: string): void {
```

Comment:
I don't mind the signature change but this is now asymmetric to `has` which isn't nice